src/jsx.d.ts

```diff
@@ -1613,6 +1613,9 @@ export namespace JSXInternal {
 
 	interface ButtonHTMLAttributes<T extends EventTarget = HTMLButtonElement>
 		extends HTMLAttributes<T> {
+		command?: Signalish<string | undefined>;
+		commandfor?: Signalist<string | undefined>;
+		commandFor?: Signalist<string | undefined>;
```

Comment:
Not sure why our `.d.ts` files aren't type checked, we have something that seems to poison the ts lang server

```suggestion
		commandfor?: Signalish<string | undefined>;
		commandFor?: Signalish<string | undefined>;
```